content/en/docs/concepts/scheduling-eviction/mutable-pod-scheduling-directives.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Pod Scheduling Readiness
```

Comment:
we don't need a page just for this. You can add a section to https://kubernetes.io/docs/concepts/scheduling-eviction/pod-scheduling-readiness/